### ISSUE TITLE
java.lang.NoSuchMethodError when Adam enabled in kts

### ISSUE THREAD

**Issue by tagantroy** [MarathonLabs/marathon#447]
**Describe the bug**
Gradle plugin is broken. 

**To Reproduce**
1. Update sample app configuration
```
marathon {
    vendor = com.malinskiy.marathon.android.VendorType.ADAM // <-- add this to sample app  
    instrumentationArgs {
        put("debug", "false")
    }
}
```
2. Run `./gradlew marathonDebugAndroidTest`

**Expected behavior**
Run marathon with Adam vendor

**Logs and reports**
```
Output: /home/ivanbalaksha/work/marathon/sample/android-app/app/build/reports/marathon/debugAndroidTest
Ignore failures: false
java.lang.NoSuchMethodError: kotlin.jvm.internal.MutablePropertyReference1Impl.<init>(Ljava/lang/Class;Ljava/lang/String;Ljava/lang/String;I)V
Finish BugSnag
```

**Additional context**
Not 100% sure, but I think it is caused by incompatibility between Kotlin 1.4(used in Marathon) and Kotlin version in gradle kts.

**Comment by simonpoole** [comment]
This "goes away" / is fixed if you use gradle 6.8.

**Comment by Malinskiy** [comment]
For your particular setup using 6.8 solves the problem, however it is not just a compatibility matrix since every build's classpath is unique

**Comment by Malinskiy** [comment]
Closed in 0.7.0 by #573
